pkg/apis/metrics/clickhouse_fetcher.go

```diff
@@ -122,6 +116,14 @@ const (
 		WHERE is_done = 0 
 		GROUP BY database, table
 	`
+
```

Comment:
@alz there is main part of backport
which affects
following picture
![image](https://user-images.githubusercontent.com/105560/96276918-0aee7e80-0fed-11eb-8909-86167fc280e2.png)